task5/src/main/java/com/clouway/adapter/http/BankOperationHandler.java

```diff
@@ -54,13 +64,17 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 
     if (operation.equalsIgnoreCase("Deposit")) {
       fundsRepository.deposit(amountAsDouble, email);
+      date = new Date();
+      fundsRepository.updateHistory(dateFormat.format(date), email, operation, amountAsDouble);
```

Comment:
What will happen if you do not format the date and store it in DB. How it is stored?